include/mbedtls/ssl.h

```diff
@@ -1795,7 +1805,7 @@ struct mbedtls_ssl_context {
                                                   *   (the end is marked by in_len).   */
 #endif /* MBEDTLS_SSL_DTLS_CONNECTION_ID */
     unsigned char *MBEDTLS_PRIVATE(in_len);      /*!< two-bytes message length field   */
-    unsigned char *MBEDTLS_PRIVATE(in_iv);       /*!< ivlen-byte IV                    */
+    unsigned char *MBEDTLS_PRIVATE(in_ext);      /*!< extension structure (ABI compat) */
```

Comment:
For the record, I am approving the design using an extension structure to keep the same general structure of the patch but preserve the ABI.

(This is not an approval on GitHub because I haven't done a full code review.)